bundled/tool/lsp_server.py

```diff
@@ -299,6 +299,63 @@ def organize_imports(
     ]
 
 
+@QUICK_FIXES.quick_fix(
+    codes=[
+        "W1406:redundant-u-string-prefix",
+        "W1401:anomalous-backslash-in-string",
+    ]
+)
+def fix_string_prefix_and_backslash(
+    document: workspace.Document, diagnostics: List[lsp.Diagnostic]
+) -> List[lsp.CodeAction]:
+    """Provides quick fixes for W1406 and W1401."""
+    actions = []
+
+    for diagnostic in diagnostics:
+        if diagnostic.code == "W1406":
+            actions.append(
+                lsp.CodeAction(
+                    "Remove redundant 'u' prefix from string",
+                    kind=lsp.CodeActionKind.QuickFix,
+                    diagnostics=[diagnostic],
+                    edit=lsp.WorkspaceEdit(
+                        changes={
+                            document.uri: [
+                                lsp.TextEdit(
+                                    diagnostic.range,
+                                    diagnostic.message.replace("u'", "'")
```

Comment:
This might not be the right fix. It is better to read the line from `document.source` or `document.lines` and generate the corrected line.

Reply:
did not mean to open PR yet, setting to draft